deploy/helm/sumologic/values.yaml

```diff
@@ -3163,8 +3157,8 @@ otelcol:
               - from: build_hostname
           source/containers:
             collector: '{{ .Values.sumologic.collectorName | default .Values.sumologic.clusterName | quote }}'
-            source_name: '{{ .Values.fluentd.logs.containers.sourceName | quote }}'
-            source_category: '{{ .Values.fluentd.logs.containers.sourceCategory | quote }}'
+            source_name: "%{k8s.namespace.name}.%{k8s.pod.name}.%{k8s.container.name}"
```

Comment:
we could use helm function to translate attributes :trollface:
seriously :thinking: